### ISSUE TITLE
WCAG 2.1: 'Switcher' drop down's inner background color has insufficient contrast against its adjacent background

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-web#761]
## Describe the bug

Related to [WCAG SC 1.4.11: Non-text Contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast)

>For active controls such as buttons and form fields: any visual information provided that is necessary for a user to identify that a control is present and how to operate it must have a minimum 3:1 contrast ratio with the adjacent colors

**To Reproduce**

1. Run assessment on any page
2. Open details view
3. Observe that the ```FastPass/Assessment``` switcher's background isn't having sufficient contrast against the adjacent background

![image](https://user-images.githubusercontent.com/22068434/59231300-0adaa500-8b95-11e9-95d4-471c081c8b30.png)

## Expected behavior

The contrast ratio between the switcher's background and the adjacent background should be at least 3:1

## Screenshots

![image](https://user-images.githubusercontent.com/22068434/59231381-4d03e680-8b95-11e9-9f51-416e113cb612.png)

## Extension

- Accessibility Insights for Web - prod
- Browser Version: Chrome 75.0.3770.80
- Target Page: Any page

## Did you search for similar existing issues?
Yes

**Comment by cheade** [comment]
Working on solutions for this - it's on my radar :)

**Comment by cheade** [comment]
Suggested treatment for context switcher:

![image](https://user-images.githubusercontent.com/40274682/59541909-44265400-8eb8-11e9-9f0f-dfc4542b9084.png)

This will put our drop-down context control in a similar visual style to our secondary buttons (i.e; user sees an icon with text outlined = clickable)

I also have another visual styled I played around with. Probably out of scope, but still an option ;):

![image](https://user-images.githubusercontent.com/40274682/59541970-95364800-8eb8-11e9-8faf-e848d70317f6.png)

**Comment by ferBonnin** [comment]
Thanks Chelsea!

**Comment by ferBonnin** [comment]
per conversation with Peter, lets go with opt1, opt 2 feels like two different controls.

**Comment by ferBonnin** [comment]
also note that need to take care that we keep focus indicator.

**Comment by cheade** [comment]
@ferBonnin for the focus indicator, can we use the same focus that we’ll have for our buttons? The internal dotted line?

**Comment by ferBonnin** [comment]
we will assume yes for now, but buttons will need to have a full feature. So if that's the case, I will attach this bug to that feature instead

**Comment by smoralesd** [comment]
@iamrafan This has been address and it should be on canary presently

**Comment by iamrafan** [comment]
Validated the fix in prod version 2.15.0
